server/Settings.php

```diff
@@ -98,8 +98,7 @@ public function setAccessToken($token): bool
             $token['expires_at'] = time() + (int) $token['expires_in'];
         }
 
-        $summarizer = (new IndexSiteSummary($this->main));
-        $summarizer->scheduleNow('autoUpdate');
+        do_action('agentwp_set_access_token', []);
```

Comment:
@galatanovidiu perhaps add the access token data as an argument in case it's needed in other hooks

Reply:
I would prefer not to expose that data to the entire ecosystem. That action can be accessed by any plugin, theme ... 
That is why I did not add it there and only an empty array